### DISCUSSION TITLE
defaultLocale

### DISCUSSION THREAD

**Discussion by QPRocky** [i18next/next-i18next#1395] [Q&A] [answered]
Here is my next-i18next.config.js

module.exports = {
  i18n: {
    locales: ["fi", "sv", "en"],
    defaultLocale: "fi",
  },
}

When I navigate to www.mydomain.com, url turns to www.mydomain.com/en instead of "fi"

What is purpose of "defaultLocale"?

**Comment by TrongTrinh** [comment]
I also get this issue, have you fixed it? 
Thanks!

**Comment by isaachinman** [comment, marked as answer]
Please check the NextJs documentation for answers about the `i18n` object within `next.config.js`. Good luck!
